llvm/test/CodeGen/AMDGPU/atomic_optimization_split_dt_update.ll

```diff
@@ -0,0 +1,153 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -O3 -mtriple=amdgcn-- -mcpu=gfx908 %s -o - -verify-dom-info | FileCheck %s
```

Comment:
I'm wondering if I should add a `-stop-before=amdgpu-atomic-optimizer` line here to make sure it runs?

I wanted to make an IR test but I couldn't reproduce the crash I experienced using opt, only with llc -stop-after but with llc I can't use update_test_checks as the IR is printed inside the MIR.

Reply:
We can invoke this pass using opt and verify dom-tree using -passes='amdgpu-atomic-optimizer,verify<domtree>' ?

Reply:
`-passes="amdgpu-atomic-optimizer,verify<domtree>"` works for me.

Reply:
Thanks, with the opt syntax it works